style.css

```diff
@@ -0,0 +1,675 @@
+  flex-direction: column;
+  background-color: #414246;
+  padding-top: 30px;
+  padding-bottom: 30px;
+  align-items: center;
+  overflow: auto;
+}
+
+.partners .partners-heading {
+  color: white;
+  font-family: "Lato", sans-serif;
+  text-align: center;
+  margin-bottom: 12px;
+}
+
+.partners .partner-icons-list {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  list-style-type: none;
+}
+
+.partners .partner-icons-list .partner-icons-list-child {
+  display: flex;
+  flex-direction: row;
+  list-style-type: none;
+  margin-top: 30px;
+}
+
+.footer {
+  background-color: #fff;
+  padding: 60px 80px;
+  display: flex;
+  flex-direction: row;
+}
+
+.footer .copyright {
+  text-align: left;
+  height: 24px;
+  font-family: 'Lato', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 12px;
+  line-height: 24px;
+  margin-left: 20px;
+  margin-top: 30px;
+}
+
+@media screen and (min-width: 768px) {
+  .navbar {
+    background-color: white;
+  }
+
+  #toggle-icon {
+    display: none;
+  }
+
+  .navbar .logo-container {
+    margin-left: 15px;
+    display: block;
+  }
+
+  .navbar .logo-container img {
+    width: 50px;
+  }
+
+  .links-container ul {
+    display: flex;
+    flex-direction: row;
+    list-style-type: none;
+    padding-top: 20px;
+    padding-right: 70px;
+  }
+
+  .links-container ul li {
+    margin-left: 45px;
+    cursor: pointer;
+  }
+
+  .header,
+  .main-program,
+  .professors-list {
+    padding-left: 150px;
+    padding-right: 150px;
+  }
+
+  .main-program .programs-list {
+    display: grid;
+    grid-template-columns: repeat(5, 175px);
+  }
+
+  .main-program .programs-list .program-container {
+    display: flex;
+    flex-direction: column;
+    margin-left: 20px;
+    min-height: 200px;
+    align-items: center;
+    padding: 10px 10px;
+  }
+
+  .footer .footer-logo-container img {
+    width: 180px;
+  }
+
+  .partners .partner-icons-list li img {
+    width: 150px;
+  }
+
+  .main-program .programs-list .program-container img {
+    width: 35px;
+  }
+
+  .main-program .programs-list .program-container .program-title {
+    margin-top: 15px;
+    text-align: justify;
+    max-width: 100px;
+    height: 20px;
+  }
+
+  .main-program .programs-list .program-container .program-desc {
+    margin-top: 25px;
+    text-align: start;
+    max-width: 120px;
+  }
+
+  .featured-professors .professors-list {
+    align-self: center;
+    display: grid;
+    grid-template-columns: repeat(2, 660px);
+  }
+
+  .featured-professors .professors-list .professor {
+    max-width: 440px;
+  }
+
+  .past-summits-sections .past-summits-images-container {
+    flex-direction: row;
+  }
+
+  .past-summits-images-container .img-container {
+    width: 400px;
+  }
+
+  .partners .partner-icons-list {
+    flex-direction: row;
+  }
+
+  .footer {
+    background-color: #fff;
+    padding: 60px 80px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+  }
```

Comment:
- [ ] Kindly make sure all `footer` contents are aligned in the center and make sure to reduce the gap between in order to match with the provided Behance design as much as possible.

   - ### YOUR DESIGN
   
    ![image](https://user-images.githubusercontent.com/87197412/225584478-8fdcc5ab-397e-443a-a241-9badc15d229e.png)

   - ### BEHANCE DESIGN
   
    ![image](https://user-images.githubusercontent.com/87197412/225584388-da48ad24-114d-4de5-906d-80cc6c29c583.png)